src/common/event-utils/index.js

```diff
@@ -11,6 +11,19 @@ function handleKeydown(keyCodes, e, callback) {
     }
 }
 
+// inverse of found keys
+function handleInverseKeydown(keyCodes, e, callback) {
```

Comment:
This is not a huge deal, but I feel that this function should be called something like: `handleNotKeydown`, since keys other than keydown keys are not actually the inverse or opposite of the key down keys.